try {
    Obj.a;
} catch (erreur) {
    console.log("a pas trouv√© la variable", erreur)
} finally {
    console.log("bah tampis !!!")
}